onnxruntime/core/providers/cpu/ml/svmclassifier.cc

```diff
@@ -125,120 +153,71 @@ Status SVMClassifier<T>::Compute(OpKernelContext* ctx) const {
 
           int64_t pos1 = (vector_count_) * (j - 1);
           int64_t pos2 = (vector_count_) * (i);
-          for (int64_t m = 0; m < class_i_support_count; m++) {
-            float val1 = coefficients_[pos1 + start_index_i + m];
-            float val2 = kernels[start_index_i + m];
-            sum += val1 * val2;
-          }
-          for (int64_t m = 0; m < class_j_support_count; m++) {
-            float val1 = coefficients_[pos2 + start_index_j + m];
-            float val2 = kernels[start_index_j + m];
-            sum += val1 * val2;
-          }
+          float* val1 = (float*)&(coefficients_[pos1 + start_index_i]);
```

Comment:
>&(coefficients_[pos1 + start_index_i]) [](start = 32, length = 38)

It'd nice to use `reinterpret_cast<float*>(coefficients_ + pos1 + start_index_i)`.

Reply:
removed